src/storage/keyvalue/JsonResourceStorage.ts

```diff
@@ -63,15 +64,69 @@ export class JsonResourceStorage<T> implements KeyValueStorage<string, T> {
     }
   }
 
-  public entries(): never {
-    // There is no way of knowing which resources were added, or we should keep track in an index file
```

Comment:
So that was a lie 😡

Reply:
Should have added a line saying the class now assumes it has ownership over all data found in the given container 😉